MvvmCross/Droid/Shared/Presenter/MvxFragmentsPresenter.cs

```diff
@@ -67,6 +67,11 @@ protected virtual void ShowFragment(MvxViewModelRequest request)
             var serializedRequest = Serializer.Serializer.SerializeObject(request);
             bundle.PutString(ViewModelRequestBundleKey, serializedRequest);
 
+            if (request is MvxViewModelInstanceRequest)
+            {
+                Mvx.Resolve<IMvxChildViewModelCache>().Cache(((MvxViewModelInstanceRequest)request).ViewModelInstance);
```

Comment:
In this case it is. But other then that `IMvxChildViewModelCache` is also used for Fragment lifecycle situations.